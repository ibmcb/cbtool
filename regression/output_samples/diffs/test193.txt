Background operation execution success. Operation "vmc-attach" will be processed asynchronously, through the command "/home/mrhines/cbtool/lib/operations//../../cbact --procid=TEST_mrhines --osp=port:6379,experiment_inst:TEST_mrhines,host:172.16.1.222,pid:TEST_mrhines,dbid:0,timout:70,protocol:TCP,kind:redis,username:mrhines --msp=cloudoption_testcloud_host:172.16.0.254,timeout:240,cloudoption_testcloud_usage:shared,host:172.16.1.222,port:27017,pid:TEST_mrhines,database:metrics,protocol:TCP,kind:mongodb,username:mrhines --oop=TESTCLOUD,simzone_d,update_attempts=156+_*update_frequency=6 --operation=vmc-attach --cn=TESTCLOUD --uuid=843F9DD3-ADF1-5693-84BB-4DC507D3FFCC --daemon". The process id is 10405.
|simzone_a               |5               |sut                     |simzone_a                       |184.153.20.164   
|simzone_b               |5               |sut                     |simzone_b                       |122.75.128.249   
|simzone_c               |5               |sut                     |simzone_c                       |200.51.186.156   
|simzone_d               |5               |lg                      |simzone_d                       |146.143.223.186  
|Used Memory                                          |18.63M                        
|Uptime (in seconds)                                  |2516353                       
|Total Connections Received                           |185597                        
|Total Commands Processed                             |11548930                      
|Storage Size                                         |18292736                      
|Data Size                                            |6517716                       
|Index Size                                           |2044000                       
|Average Object Size                                  |616.449068382                 
|CB82AC01-D297-5670-8658-34447B4002D4 (simzone_a) VM RESERVATIONS|0                             
|E6CE5882-73C8-5912-A1D2-D4818AE2334F (simzone_b) VM RESERVATIONS|0                             
|843F9DD3-ADF1-5693-84BB-4DC507D3FFCC (simzone_d) VM RESERVATIONS|0                             
|554D576D-8116-5E8D-B6D5-AEFA6C4C8469 (simzone_c) VM RESERVATIONS|0                             
