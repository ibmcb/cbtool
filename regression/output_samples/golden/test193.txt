Background operation execution success. Operation "vmc-attach" will be processed asynchronously, through the command "/home/mrhines/cbtool/lib/operations//../../cbact --procid=TEST_mrhines --osp=port:6379,experiment_inst:TEST_mrhines,host:172.16.1.222,pid:TEST_mrhines,dbid:0,timout:70,protocol:TCP,kind:redis,username:mrhines --msp=cloudoption_testcloud_host:172.16.0.254,timeout:240,cloudoption_testcloud_usage:shared,host:172.16.1.222,port:27017,pid:TEST_mrhines,database:metrics,protocol:TCP,kind:mongodb,username:mrhines --oop=TESTCLOUD,simzone_d,update_attempts=156+_*update_frequency=6 --operation=vmc-attach --cn=TESTCLOUD --uuid=905BFF26-22E2-50BE-9DF2-9EC73443EBFC --daemon". The process id is 26647.

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
The following VMCs are attached to this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |5               |sut                     |simzone_a                       |21.246.1.190     
|simzone_b               |5               |sut                     |simzone_b                       |27.115.158.211   
|simzone_c               |5               |sut                     |simzone_c                       |178.143.117.234  
|simzone_d               |5               |lg                      |simzone_d                       |179.8.30.251     

The VMC object simzone_d, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |156                                  
|update_frequency                         |6                                    

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |18.70M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2511640                       
|Total Connections Received                           |185319                        
|Total Commands Processed                             |11029074                      
|Number of Keys                                       |245                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18264064                      
|Data Size                                            |5776884                       
|Index Size                                           |1864128                       
|Average Object Size                                  |622.643242078                 
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |4                             
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |0                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |4                             

 ---------------- VMC-WIDE COUNTERS ----------------
|DE8A62CD-5C07-5496-9055-32EC4E525C1B (simzone_b) VM RESERVATIONS|0                             
|A631B967-7C7E-5A5D-B63E-51A88AF8A16B (simzone_a) VM RESERVATIONS|0                             
|39F9C66C-D7C3-5A50-9B41-9F9120360E56 (simzone_c) VM RESERVATIONS|0                             
|905BFF26-22E2-50BE-9DF2-9EC73443EBFC (simzone_d) VM RESERVATIONS|0                             

